docs/en/operations/settings/settings.md

```diff
@@ -1512,6 +1512,26 @@ Possible values:
 
 Default value: `0`.
 
+## query_cache_max_size {#query-cache-max-size}
+
+The maximum amount of memory (in bytes) the current user may allocate in the query cache.
+
+Possible values:
+
+- Positive integer >= 0.
+
+Default value: UINT64_MAX (no restriction).
```

Comment:
Hm, that's right ... most other settings indeed assign `0` a special meaning as "unlimited". I'll do the same here. If someone wants to disable cache usage for a specific user, then there are other means instead of setting `query_cache_max_size`/`query_cache_max_items` to 0 (specifically setting `enable_writes_from_query_cache` to false and making it read-only).